### PR TITLE
Add readme files

### PR DESCRIPTION
**Pull request by naiteeks**

### CODE REVIEW

File: edge-modules/extensions/customvision/readme.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Custom Vision
+
```
Comment (anilmur) [inline edge-modules/extensions/customvision/readme.md:2]: ... can deploy an AI inference service on the edge for detecting...

File: edge-modules/extensions/customvision/readme.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+[Azure Custom Vision](https://docs.microsoft.com/azure/cognitive-services/custom-vision-service/overview) is an image recognition service that lets you build, deploy, and improve your own image identifiers. Using the [AI extensibility capability](https://docs.microsoft.com/azure/azure-video-analyzer/video-analyzer-docs/analyze-live-video-without-recording#analyzing-video-using-a-custom-vision-model) of Video Analyzer we can deploy a model on the edge for detecting objects such as toy trucks from a live video stream.
+
+## Running and testing
+A full fledged [tutorial](https://aka.ms/ava-custom-vision) is available to learn more about how to use this module.
```
Comment (anilmur) [inline edge-modules/extensions/customvision/readme.md:6]: ... IoT Edge module.

File: edge-modules/extensions/intel/readme.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Intel module extensions to be used with Azure Video Analyzer
+
+This folder contains a set of IoT Edge module extensions from Intel that can be used in conjunction with Azure Video Analyzer
```
Comment (anilmur) [inline edge-modules/extensions/intel/readme.md:3]: ...contains a set of IoT Edge modules from Intel... Analyzer.

The modules act as AI inference servers. They implement the Video Analyzer [gRPC extension protocol](???) and [HTTP extension protocol](???). This enables you to implement live video workflows, shown in tutorials [here](???) and [here](???).

File: edge-modules/extensions/readme.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# IoT Edge module extensions to be used with Azure Video Analyzer
+
+This folder contains a set of IoT Edge module extensions that can be used in conjunction with Azure Video Analyzer
```
Comment (anilmur) [inline edge-modules/extensions/readme.md:3]: ...IoT Edge module [extensions](pipeline-extensions.md - but use docs.msft.com)...

File: edge-modules/extensions/readme.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+## Contents
+
+| Folders | Description |
+|---------|-------------|
+|customvision|Docker container to build a custom vision model |
+|intel|Docker container to build Intel extension modules |
```
Comment (anilmur) [inline edge-modules/extensions/readme.md:10]: containers

File: edge-modules/extensions/readme.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+|yolo|Docker container to build yolov modules|
```
Comment (anilmur) [inline edge-modules/extensions/readme.md:11]: containers

File: edge-modules/sources/readme.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# Video sources to be used with Azure Video Analyzer
+
+This folder contains a set of docker containers that can be used in conjunction with Azure Video Analyzer
```
Comment (anilmur) [inline edge-modules/sources/readme.md:3]: ... a set of docker containers that can simulate a live video source for use with Azure Video Analyzer.

**Review by anilmur — APPROVED** [review]
Minor edits

**Comment by naiteeks** [comment]
@gadamilan  - This PR is good to merge

**Pull request merged**